src/Illuminate/Support/Collection.php

```diff
@@ -1322,4 +1322,19 @@ protected function getArrayableItems($items)
 
         return (array) $items;
     }
+
+    /**
+     * Drop the first or last {$limit} items.
```

Comment:
@GrahamCampbell the comment is copied from the `take` method. Should both be changed?

If you prefer that comment I change it :smiley: